sympy/printing/tests/test_repr.py

```diff
@@ -101,12 +101,11 @@ def test_Rational():
 
 
 def test_Float():
-    sT(Float('1.23', prec=3), "Float('1.22998', prec=3)")
```

Comment:
I think these tests should stay almost "as is", just with dps=.  Then you can add your new tests as you've done here.  Logic here is nothing wrong with the old tests so leave then alone as Mich as possible .